DiffusionPreprocessing/scripts/eddy_postproc.sh

```diff
@@ -17,6 +17,18 @@ globalscriptsdir=${HCPPIPEDIR_Global}
 eddydir=${workingdir}/eddy
 datadir=${workingdir}/data
 
+echo "Generating eddy QC report in ${eddy_dir}/eddy_unwarped_images.qc"
+qc_command="${FSLDIR}/bin/eddy_quad "
+qc_command+="${eddydir}/eddy_unwarped_images "
+qc_command+="-idx ${eddydir}/index.txt "
+qc_command+="-par ${eddydir}/acqparams.txt "
+qc_command+="-m ${eddydir}/nodif_brain_mask.nii.gz "
+qc_command+="-b ${eddydir}/Pos_Neg.bvals "
+qc_command+="-g ${eddydir}/Pos_Neg.bvecs "
```

Comment:
What does the `-g` flag do?  I can't tell from the QUAD UserGuide: https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/eddyqc/UsersGuide.  In particular, what if `eddy` isn't run with the `--residuals` option?  Does QUAD then fail if `-g` is specified?

Also, what is the role of the `-s, --slspec` flags in QUAD?  Should we be including that?  And if so, is there an option to read the json instead?

Reply:
The plan is to call `eddy` using `--extra-eddy-arg=--json=<jsonfile.json>`.  So since, we will never need to construct the `slspec` file, I guess we wouldn't have whatever additional info the `slspec` approach can approach.  Are we missing anything substantive if we don't have that?

Reply:
I think QUAD just adds a slice2volume summary measure to the tables in the single-subject PDF report. The distribution of this summary measure would then also be included in the group report by SQUAD. So I don't think it would be too crucial to include (my guess would be that this slice2volume metric would correlate quite strongly with the general movement metrics anyway, but I have no evidence for that).

Reply:
I know you said that the `-g` flag does nothing currently, but for the future (when it does something) shouldn't we be using `${eddydir}/eddy_unwarped_images.eddy_rotated_bvecs` as the argument to `-g`?

Relatedly, in this section of `eddy_postproc.sh`:
```
if [ ${CombineDataFlag} -eq 2 ]; then
	${FSLDIR}/bin/imcp  ${eddydir}/eddy_unwarped_images ${datadir}/data
	cp ${eddydir}/Pos_Neg.bvals ${datadir}/bvals
	cp ${eddydir}/Pos_Neg.bvecs ${datadir}/bvecs
```
shouldn't we instead be doing:
```
if [ ${CombineDataFlag} -eq 2 ]; then
	${FSLDIR}/bin/imcp  ${eddydir}/eddy_unwarped_images ${datadir}/data
	cp ${eddydir}/Pos_Neg.bvals ${datadir}/bvals
	cp ${eddydir}/Pos_Neg.bvecs ${datadir}/bvecs_noRot
        cp ${eddydir}/eddy_unwarped_images.eddy_rotated_bvecs ${datadir}/bvecs
```

Reply:
Fixed